Mathlib/Algebra/Module/Torsion.lean

```diff
@@ -591,6 +591,19 @@ instance (a : R) {S : Type*} [SMul S R] [SMul S M] [IsScalarTower S R M] [IsScal
     IsScalarTower S (R ⧸ R ∙ a) (torsionBy R M a) :=
   inferInstance
 
+/-- Given an `R`-module `M` and an element `a` in `R`, submodules of the `a`-torsion submodule of
+`M` do not depend on whether we take scalars to be `R` or `R ⧸ R ∙ a`. -/
+def submodule_torsionBy_orderIso (a : R) :
```

Comment:
```suggestion
@[simps]
def submodule_torsionBy_orderIso (a : R) :
```

Reply:
Sadly no.

I would have to state the LHS using `R ⧸ Ideal.span {a}` rather than `R ⧸ R ∙ a` to appease the linter. These are defeq but I much prefer the latter because of the notation, and besides all the adjacent lemmas use this spelling.

I think the right fix is probably to drop `simp` from `Ideal.submodule_span_eq` and make `Ideal.span` an `abbrev`.

Reply:
Ok, thanks for debugging. Should we record this in an issue, or as a comment in this/that file?

Reply:
Opinions might be divided so I'll start a thread on Zulip.